### ISSUE TITLE
Window compile fail

### ISSUE THREAD

**Issue by JeongyongShin** [cpvrlab/ImagePlay#116]
Building according to the guide given in the window environment.
If you perform the command in Git Bash, how do I fix the error?

`kwizm@LAPTOP-DE3PEO1B MINGW64 ~/ImagePlay (master)
$ qmake -tp vc -r

Qt: Untested Windows version 10.0 detected!
Reading C:/Users/kwizm/ImagePlay/IPL/IPL.pro
Reading C:/Users/kwizm/ImagePlay/ImagePlay/ImagePlay.pro
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
Qt: Untested Windows version 10.0 detected!
WARNING: Could not parse Compiler option '-FS'; added to AdditionalOptions.
WARNING: You can suppress these warnings with CONFIG+=suppress_vcproj_warnings.
WARNING: Could not parse Compiler option '-Zc:strictStrings'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-FS'; added to AdditionalOptions.
Project WARNING: Fervor: falling back to application name 'ImagePlay'
Project WARNING: Fervor: falling back to application version '6.1.0'
Project MESSAGE: Defines:
Project MESSAGE: UNICODE WIN32 WIN64 APP_VERSION=\"6.1.0\" APP_COMPANY=\"BFH\" APP_PRODUCT=\"ImagePlay\" APP_DESCRIPTION=\"\" APP_COPYRIGHT=\"\" APP_NAME=\"ImagePlay\" IMAGEPLAY_URL=\"http://imageplay.io\" IMAGEPLAY_APPCAST_URL=\"http://imageplay.io/Appcast.xml\" FV_APP_NAME=\"ImagePlay\" FV_APP_VERSION=\"6.1.0\" USE_FERVOR_UPDATER
Project WARNING: Fervor: falling back to application name 'ImagePlay'
Project WARNING: Fervor: falling back to application version '6.1.0'
Project MESSAGE: Defines:
Project MESSAGE: UNICODE WIN32 WIN64 APP_VERSION=\"6.1.0\" APP_COMPANY=\"BFH\" APP_PRODUCT=\"ImagePlay\" APP_DESCRIPTION=\"\" APP_COPYRIGHT=\"\" APP_NAME=\"ImagePlay\" IMAGEPLAY_URL=\"http://imageplay.io\" IMAGEPLAY_APPCAST_URL=\"http://imageplay.io/Appcast.xml\" FV_APP_NAME=\"ImagePlay\" FV_APP_VERSION=\"6.1.0\" USE_FERVOR_UPDATER
Project WARNING: Fervor: falling back to application name 'ImagePlay'
Project WARNING: Fervor: falling back to application version '6.1.0'
Project MESSAGE: Defines:
Project MESSAGE: UNICODE WIN32 WIN64 APP_VERSION=\"6.1.0\" APP_COMPANY=\"BFH\" APP_PRODUCT=\"ImagePlay\" APP_DESCRIPTION=\"\" APP_COPYRIGHT=\"\" APP_NAME=\"ImagePlay\" IMAGEPLAY_URL=\"http://imageplay.io\" IMAGEPLAY_APPCAST_URL=\"http://imageplay.io/Appcast.xml\" FV_APP_NAME=\"ImagePlay\" FV_APP_VERSION=\"6.1.0\" USE_FERVOR_UPDATER
WARNING: Could not parse Compiler option '-FS'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-Zc:strictStrings'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-FS'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-FS'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-Zc:strictStrings'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-FS'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-FS'; added to AdditionalOptions.
WARNING: Could not parse Compiler option '-Zc:strictStrings'; added to AdditionalOptions.
Project WARNING: Fervor: falling back to application name 'ImagePlay'
Project WARNING: Fervor: falling back to application version '6.1.0'
Project MESSAGE: Defines:
Project MESSAGE: UNICODE WIN32 WIN64 APP_VERSION=\"6.1.0\" APP_COMPANY=\"BFH\" APP_PRODUCT=\"ImagePlay\" APP_DESCRIPTION=\"\" APP_COPYRIGHT=\"\" APP_NAME=\"ImagePlay\" IMAGEPLAY_URL=\"http://imageplay.io\" IMAGEPLAY_APPCAST_URL=\"http://imageplay.io/Appcast.xml\" FV_APP_NAME=\"ImagePlay\" FV_APP_VERSION=\"6.1.0\" USE_FERVOR_UPDATER
`

**Comment by drzraf** [comment]
These are warnings, but I don't see actual "errors".

**Comment by JeongyongShin** [comment]
> These are warnings, but I don't see actual "errors".

Yeah, well, I don't think there should be any problem with running this Warning in Visual Studio. There were countless errors composing in Visual Studio. I'll upload it separately. Please review it.
